src/main.rs

```diff
@@ -155,7 +155,7 @@ fn changelog_print(old: &cargo::core::Package, new: &cargo::core::Package) -> Re
     let old = get_changelog(old.root())?;
     let new = get_changelog(new.root())?;
     let diff = changelog_diff(old, new);
-    if diff != "" {
+    if !diff.is_empty() {
```

Comment:
What kind of change is this? A stylistic one? Or does it generate different code? Is there a performance difference?

Reply:
No, the new one just doesn't provoke clippy to complain :-D. It's equivalent (clippy says it's more readable).